assignments/P2.md

```diff
@@ -48,7 +48,7 @@ and adds a makefile `username/P2/Makefile` to build `solution.c` into an executa
 0. Copy the driver code and makefile into your `ctf/` folder
 
     * Examine the driver code to understand how information flows from one function to another
-
```

Comment:
I dunno how you ended up with a merge commit, but it should go. Make sure to check out a branch based on 2024_fall_uml to put your commits on top and then make the PR from that branch in your fork to the 2024_fall_uml branch here in order to avoid the mess.